db/seeds.rb

```diff
@@ -30,3 +30,18 @@
 ].each do |frequency|
   LeaseFrequency.where(frequency: frequency).first_or_create
 end
+
+[
+  {name: "Happy User", email: "user@happysoftware.dev", password_digest: "$2a$12$9flOWQYOUEaQjwElT6eayeUegdkm9vMhDUEl1N.mAxFGCYVZ/ngY.", activated: true, activated_at: Date.parse("2022-01-01") }
```

Comment:
This `password_digest` is super annoying.
Will have to figure out a way to `first_or_create` a user.

This seed will only work the first time it is run, once the record is created, validations on the user model fails and `first_or_create` returns nothing for this user.

Idk why - it's a future problem to deal with.

For now, I just want something to create records in the db when the db gets wiped instead of me having to manually create everything.